src/topkg_build.ml

```diff
@@ -33,11 +38,16 @@ let with_dir b dir = { b with dir }
 let nop = fun _ -> Ok ()
 
 let cmd c os files =
-  Topkg_os.Cmd.run @@ Topkg_cmd.(build_cmd c os %% of_list files)
+  Topkg_os.Cmd.run @@
+    Topkg_cmd.(build_cmd ~use_toolchain:false c os %% v "-just-plugin") >>= fun () ->
+  Topkg_os.Cmd.run @@
+    Topkg_cmd.(build_cmd ~use_toolchain:true c os %% of_list files)
```

Comment:
I'm not very fond of this two level invocation here, in particular the docs [here](http://erratique.ch/software/topkg/doc/Topkg.Pkg.html#VALbuild) will need update. Is there a valid use case why `ocamlbuild` would use the `toolchain` argument to build its own plugin ? Shouldn't `ocamlbuild`'s toolchain option simply affect the requested targets ?